### PR TITLE
[Task #4] Completed the Update CSV method

### PR DESCRIPTION
**Pull request by nashawnch**
## Description

Completed the Update CSV method

## Type of change

- [X] New feature (non-breaking change which adds functionality)

## Checklist:

- [X] Title is in "[Task #] Description" format
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: minimax.py [hunk 1]
```diff
@@ -289,10 +290,27 @@ def create_players(player_csv):
...
-    # TODO @Nashawn: update player.csv 
-    # player_lookup holds {lowercased_name, player_object}
-    pass
+# This method takes in the player object and updates the CSV with the latest data from the object at the time of being called.
+
+def update_players_csv(player):
+    
+    newPlayerData = [player.name, player.optimal_moves , player.total_moves, player.optimal_percent]
+
+    with open('players.csv', 'r') as readFile:
```
Comment (tyreest19) [inline minimax.py]: Can you move player.csv to CSVFolder. We are placing all of our CSV files there.

File: minimax.py [hunk 2]
```diff
@@ -289,10 +290,27 @@ def create_players(player_csv):
...
+        reader = csv.reader(readFile)
+        lines = list(reader)
+        print(lines)
+        for i in range(len(lines)):
+            if lines[i][0] == player:
+                print(lines[i])
+                lines[i] = newPlayerData
+
+    with open('player.csv', 'w') as writeFile:
```
Comment (tyreest19) [inline minimax.py]: Can you move player.csv to CSVFolder. We are placing all of our CSV files there.

**Review by tyreest19 — COMMENTED** [review]
@Johnsons-AI/tic-tac Whose making the player.csv file? I add some comments on file placement other than that everything is looking great!!!

File: minimax.py (hunk 2 above)
Comment (YanceyR) [reply minimax.py]: agreed ^^

**Review by YanceyR — CHANGES_REQUESTED** [review]

**Comment by tyreest19** [comment]
I updated this branch so all player.csv calls are now CSVFolder/player.csv . Can we now merge this @Johnsons-AI/tic-tac ?

**Review by YanceyR — APPROVED** [review]
Good job Tyree!

**Pull request merged**